lms/envs/common.py

```diff
@@ -1939,3 +1939,7 @@
 
 #date format the api will be formatting the datetime values
 API_DATE_FORMAT = '%Y-%m-%d'
+
+
+############# OPEN EDX COURSE API #################
+FEATURES['COURSE_API'] = False
```

Comment:
Specifically: ask Solutions.
Generally: not all installations need the API, failsafe (since we don't have throttling)